src/app.js

```diff
@@ -38,24 +41,140 @@ gpii.app.electronAppListener = function () {
 require("electron").app.on("ready", gpii.app.electronAppListener);
 
 
+/**
+ * Responsible for creation and housekeeping of the connection to the PCP Channel WebSocket
+ */
+fluid.defaults("gpii.app.gpiiConnector", {
+    gradeNames: "fluid.modelComponent",
+
+    // Configuration regarding the socket connection
+    config: {
+        gpiiWSUrl: "ws://localhost:8081/pcpChannel"
+    },
+
+    model: {
+        socket: "@expand:gpii.app.createGPIIConnection({that}.options.config)"
+    },
+
+    events: {
+        onPreferencesUpdate: null
```

Comment:
With the "on" convention it's more normal for past tense - i.e. "onPreferencesUpdated"

Reply:
I totally agree with this, but we were influenced by the framework's event names, such as `onCreate`, `onDestroy` events.
Anyway, I'll change it to past tense